kyber/objects/secret.py

```diff
@@ -41,10 +41,7 @@ def __unicode__(self):
         return unicode(repr(self))
 
     def has_key(self, k):
-        return self.obj['data'].has_key(k)
-
-    def pop(self, k, d=None):
-        return self.obj['data'].pop(k, d)
+        return k in self.obj['data']
```

Comment:
Pop was defined twice, with the definition below, overwriting this one, being 

```
def pop(self, *args):
    return self.obj['data'].pop(*args)
```

`has_key` is deprecated and [has been removed in python3](https://docs.python.org/3.1/whatsnew/3.0.html#builtins) in favour of using `in`. Though I didn't realize the function here which used it is also `has_key()`. We can disable this rule though (the PR just has default rules at the moment). Thoughts?